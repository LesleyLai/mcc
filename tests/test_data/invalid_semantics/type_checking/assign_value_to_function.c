int main(void) {
    int x(void);
    x = 3;
    return 0;
}
