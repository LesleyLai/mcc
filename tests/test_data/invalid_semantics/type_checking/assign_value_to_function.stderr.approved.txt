{{filename}}:3:5: Error: invalid operands to binary expression ('int(void)' and 'int')
3 |     x = 3;
  |     ^~~~~

